marklogic-unit-test-client/src/main/java/com/marklogic/test/unit/JUnitTestSuite.java

```diff
@@ -9,23 +9,26 @@
  */
 public class JUnitTestSuite {
 
-	private String xml;
-	private int errors;
-	private int failures;
-	private String hostname;
-	private String name;
-	private int tests;
-	private double time;
+	private final String xml;
+	private final int errors;
+	private final int failures;
+	private final String hostname;
+	private final String name;
+	private final int tests;
+	private final double time;
+  private final int cases;
+
 	private List<JUnitTestCase> testCases;
 
-	public JUnitTestSuite(String xml, int errors, int failures, String hostname, String name, int tests, double time) {
+	public JUnitTestSuite(String xml, int errors, int failures, String hostname, String name, int cases, int tests, double time) {
 		this.xml = xml;
 		this.errors = errors;
 		this.failures = failures;
 		this.hostname = hostname;
 		this.name = name;
 		this.tests = tests;
 		this.time = time;
+    this.cases = cases;
```

Comment:
What is the difference between `tests` and `cases`? I'm thinking first that `cases` should be `testCases`, but looking at this, I don't know what the difference between the two is.

Reply:
Tests is all tests, cases is all testcases, cases should be less than tests

Reply:
We likely can't change it in a minor release, but should `tests` really be `testSuites` then? And cases is `testCases`? 

My understanding of marklogic-unit-test is there are two terms when referring to "tests":

1. A "test" refers to a file that becomes a module in the modules database, and it can have many assertions in it - e.g. `test:assert-equal`. 
2. A "suite" refers to a directory that contains one to many tests. 

I don't know of a term that captures "all suites" though. 

So basically, I'm wondering what "cases" refers to. Is it something different than a "test" or a "suite", or is it the same as one of those?